addons/website_event_track/views/event_track_templates.xml

```diff
@@ -225,15 +225,17 @@
                                 <div class="col-lg-12 text-center">
                                     <h2 t-field="track.name"/>
                                     <small>Proposed by
-                                        <span t-if="track.partner_id" t-field="track.partner_id.sudo().name"/>
-                                        <span t-if="not track.partner_id" t-field="track.partner_name"/>
+                                        <span t-if="track.partner_name" t-field="track.partner_name"/>
+                                        <span t-if="track.partner_id">
+                                            (<t t-esc="track.partner_id.sudo().name"/>)
```

Comment:
partner_name is a shortcut of partner.name or a custom name (e.g.: binome)... It is not a company or whatever